config.template.yml

```diff
@@ -582,84 +581,102 @@ notifier:
 ##
 # identity_providers:
```

Comment:
If you check this section in your code, the indent of all of oidc is broken. I'd suggest copying it from master.

Sometimes editors will want to auto correct the indent when you paste and you can undo that with CTRL-Z